presto-tests/src/test/java/com/facebook/presto/execution/TestQueues.java

```diff
@@ -281,6 +296,16 @@ public void testQueryTypeBasedSelection()
         }
     }
 
+    private void assertResourceGroupWithClientTags(DistributedQueryRunner queryRunner, Set<String> clientTags, String expectedResourceGroup)
```

Comment:
This method is very similar to `assertResourceGroup`, if you update `assertResourceGroup` to take `session` as a parameter:

```
private void assertResourceGroup(DistributedQueryRunner queryRunner, Session session, String query, String expectedResourceGroup)
```

`assertResourceGroupWithClientTags ` can be removed (then `createQueryWithClientTags` can be removed too).

e.g.,
```
assertResourceGroupWithClientTags(queryRunner, ImmutableSet.of("a"), "global.a.default");
```

will be

```
assertResourceGroup(queryRunner, newSessionWithTags(ImmutableSet.of("a")), LONG_LASTING_QUERY, "global.a.default");
```